bsmetadata/experiments/with_metadata.py

```diff
@@ -11,6 +11,9 @@
 logger = logging.getLogger(__name__)
 
 
+load_dataset = functools.partial(load_dataset, use_auth_token=True)
```

Comment:
This is something we don't necessarily need in our workflow I think as we work on the dataset already cloned locally. After that, I understand that you may need it for other tests on your side. :relaxed: